lib/Doctrine/ODM/MongoDB/Mapping/Driver/AnnotationDriver.php

```diff
@@ -150,14 +147,14 @@ public function loadMetadataForClass($className, \Doctrine\Persistence\Mapping\C
             $class->setWriteConcern($documentAnnot->writeConcern);
         }
         if (isset($documentAnnot->indexes) && count($documentAnnot->indexes)) {
-            @trigger_error(
-                sprintf(
-                    'Indexes parameter used in %s\'s %s was deprecated in doctrine/mongodb-odm 2.2 and will be removed in 3.0. Specify all Index and UniqueIndex annotations on a class level.',
-                    $className,
-                    get_class($documentAnnot)
-                ),
-                E_USER_DEPRECATED
+            trigger_deprecation(
+                'doctrine/mongodb-odm',
+                '2.1',
```

Comment:
"was deprecated in doctrine/mongodb-odm 2.2"
```suggestion
                '2.2',
```